### ISSUE TITLE
Not able to setup localhost 

### ISSUE THREAD

**Issue by morgoth** [basecamp/google_sign_in#14]
I'm trying to upgrade to version 1.0.

I'm having problem with localhost flow.
After clicking sign in button, I get the error in the browser:
```
OPTIONS https://accounts.google.com/o/oauth2/auth?client_id=654611453499-fe2riqsl6rcm3p6c0r7djp1sfk4rd2j3.apps.googleusercontent.com&prompt=login&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Fgoogle_sign_in%2Fcallback&response_type=code&scope=openid+profile+email&state=%2FEa2U95dBAhiAYDTNbkqqg%3D%3D 405
new#:1 Failed to load https://accounts.google.com/o/oauth2/auth?client_id=654611453499-fe2riqsl6rcm3p6c0r7djp1sfk4rd2j3.apps.googleusercontent.com&prompt=login&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Fgoogle_sign_in%2Fcallback&response_type=code&scope=openid+profile+email&state=%2FEa2U95dBAhiAYDTNbkqqg%3D%3D: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:3000' is therefore not allowed access.
```
When I follow this url by hand, the whole flow works correct.

@georgeclaghorn Do you have any clue what could be wrong? It looks like problem with CORS, howerver not sure how I could fix it. Note, that I already have `http://localhost:3000/google_sign_in/callback` in the `Authorized redirect URIs`

**Comment by georgeclaghorn** [comment]
Which browser are you using?

**Comment by georgeclaghorn** [comment]
I can’t reproduce in a fresh Rails app. Is it possible you have some JavaScript that converts cross-origin HTTP redirects into XHRs? (Turbolinks specifically avoids doing this.)

**Comment by morgoth** [comment]
Indeed. I have `jquery_ujs`, but it's the same problem with `rails-ujs`.
So the form generated by google_sign_in helper uses `form_with` which by default uses `ujs` to submit form "remotely". When I add `local: true` here https://github.com/basecamp/google_sign_in/blob/master/app/helpers/google_sign_in/button_helper.rb#L3 it works fine.
Can we make it somehow configurable (or disable), as currently options are not passed to the form, but only to button tag?

**Comment by georgeclaghorn** [comment]
I think we can just make the form local. We’re never going to return JS for execution or a same-origin redirect.

**Comment by morgoth** [comment]
Thank you. @georgeclaghorn can we have a release with this fix?

**Comment by georgeclaghorn** [comment]
I just published v1.0.2.
